assets/xml/overlays/ovl_Arrow_Ice.xml

```diff
@@ -1,11 +1,12 @@
 <Root>
+    <!-- Ice Arrow assets -->
     <File Name="ovl_Arrow_Ice" BaseAddress="0x80922430" RangeStart="0x980" RangeEnd="0x1DA0">
-        <Texture Name="sIceArrow1Tex" OutName="ice_tex_1" Format="ia8" Width="32" Height="64" Offset="0x980"/>
-        <Texture Name="sIceArrow2Tex" OutName="ice_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1180"/>
-        <Array Name="sIceArrowVtx" Count="43" Offset="0x1980">
+        <Texture Name="gIceArrow1Tex" OutName="ice_arrow_tex_1" Format="ia8" Width="32" Height="64" Offset="0x980"/>
+        <Texture Name="gIceArrow2Tex" OutName="ice_arrow_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1180"/>
+        <Array Name="gIceArrowVtx" Count="43" Offset="0x1980">
             <Vtx/>
         </Array>
-        <DList Name="sIceArrowDL" Offset="0x1C30"/>
-        <DList Name="sIceArrowVtxDL" Offset="0x1CE0"/>
+        <DList Name="gIceArrowMaterialDL" Offset="0x1C30"/>
+        <DList Name="gIceArrowModelDL" Offset="0x1CE0"/>
```

Comment:
Just to add to the conversation about the names being the same between OoT and MM. I too believe it's better when both names between the games are the same. But if there's a better way to name something, then using the better name in MM and seeing if OoT will change it later is also a valid option. I think having the prefix `gIceArrow` would be better for both games.

Reply:
Well, for OoT, if it's static then it should probably be `sIceArrow` as the prefix, no? Which sort of goes back to the question of "why do OoT and MM handle in-overlay assets differently?" But that's something way beyond the scope of this tiny PR.